src/test/java/org/cactoos/proc/RunnableOfTest.java

```diff
@@ -33,50 +33,70 @@
  * Test case for {@link RunnableOf}.
  *
  * @since 0.2
- * @checkstyle JavadocMethodCheck (500 lines)
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
 final class RunnableOfTest {
 
     @Test
     void convertsProcIntoRunnable() {
-        final AtomicBoolean done = new AtomicBoolean();
+        final AtomicReference<Object> done = new AtomicReference<>();
+        final Object obj = new Object();
         new Assertion<>(
             "Must execute Runnable with Proc",
             new RunnableOf(
                 new ProcOf<>(
-                    ignored -> {
-                        done.set(true);
+                    input -> {
+                        done.set(input);
                     }
                 ),
-                "ignored"
+                obj
             ),
-            new MatcherOf<Runnable>(
-                input -> {
-                    input.run();
-                    return done.get();
+            new MatcherOf<>(
+                runnable -> {
+                    runnable.run();
+                    return done.get().equals(obj);
```

Comment:
@victornoel I think that in this context, it is better to use `==` because we actually get object `obj`.